exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -83,56 +84,77 @@ public String getContent() {
+    //Select background color from palette
+    String opTblBgColor = OPERATOR_OVERVIEW_BGCOLOR_PALETTE[major%2];
     String path = new OperatorPathBuilder().setMajor(major).setOperator(firstProfile).build();
-    tb.appendCell(path, null);
-    tb.appendCell(operatorName, null);
+    tb.appendCellWithBgColor(path, null, null, opTblBgColor);
+    tb.appendCell(operatorName, null, null);
+
+    //Get MajorFragment Busy+Wait Time Tally
+    long majorBusyNanos = majorFragmentBusyTally.get(new OperatorPathBuilder().setMajor(major).build());
 
     double setupSum = 0.0;
     double processSum = 0.0;
     double waitSum = 0.0;
     double memSum = 0.0;
+    long recordSum = 0L;
     for (ImmutablePair<OperatorProfile, Integer> ip : ops) {
       OperatorProfile profile = ip.getLeft();
       setupSum += profile.getSetupNanos();
       processSum += profile.getProcessNanos();
       waitSum += profile.getWaitNanos();
       memSum += profile.getPeakLocalMemoryAllocated();
+      for (final StreamProfile sp : profile.getInputProfileList()) {
+        recordSum += sp.getRecords();
+      }
     }
 
-    final ImmutablePair<OperatorProfile, Integer> shortSetup = Collections.min(ops, Comparators.setupTime);
     final ImmutablePair<OperatorProfile, Integer> longSetup = Collections.max(ops, Comparators.setupTime);
-    tb.appendNanos(shortSetup.getLeft().getSetupNanos(), null);
-    tb.appendNanos(Math.round(setupSum / size), null);
-    tb.appendNanos(longSetup.getLeft().getSetupNanos(), null);
+    tb.appendNanos(Math.round(setupSum / size), null, null);
+    tb.appendNanos(longSetup.getLeft().getSetupNanos(), null, null);
 
-    final ImmutablePair<OperatorProfile, Integer> shortProcess = Collections.min(ops, Comparators.processTime);
     final ImmutablePair<OperatorProfile, Integer> longProcess = Collections.max(ops, Comparators.processTime);
-    tb.appendNanos(shortProcess.getLeft().getProcessNanos(), null);
-    tb.appendNanos(Math.round(processSum / size), null);
-    tb.appendNanos(longProcess.getLeft().getProcessNanos(), null);
+    tb.appendNanos(Math.round(processSum / size), null, null);
+    tb.appendNanos(longProcess.getLeft().getProcessNanos(), null, null);
 
     final ImmutablePair<OperatorProfile, Integer> shortWait = Collections.min(ops, Comparators.waitTime);
     final ImmutablePair<OperatorProfile, Integer> longWait = Collections.max(ops, Comparators.waitTime);
-    tb.appendNanos(shortWait.getLeft().getWaitNanos(), null);
-    tb.appendNanos(Math.round(waitSum / size), null);
-    tb.appendNanos(longWait.getLeft().getWaitNanos(), null);
+    tb.appendNanos(shortWait.getLeft().getWaitNanos(), null, null);
+    tb.appendNanos(Math.round(waitSum / size), null, null);
+    tb.appendNanos(longWait.getLeft().getWaitNanos(), null, null);
+
+    tb.appendPercent(processSum / majorBusyNanos, null, String.format(
+        "&#8721;OperatorBusy: %,.2fs / &#8721;MajorBusy: %,.2fs", processSum/1E9, majorBusyNanos/1E9));
```

Comment:
This was just to quantify the numbers, but it probably isn't needed any more. I think we can remove this.